com.avaloq.tools.ddk.test.ui/src/com/avaloq/tools/ddk/test/ui/swtbot/CoreSwtbotTools.java

```diff
@@ -223,15 +223,13 @@ public static void captureScreenshot(final SWTBot bot, final String prefix, fina
   public static boolean checkOpenWindow(final SWTWorkbenchBot bot, final String windowName) {
     Assert.isNotNull(bot, ARGUMENT_BOT);
     Assert.isNotNull(windowName, "windowName");
-    Boolean windowFound = false;
     try {
       final SWTBotShell shell = bot.shell(windowName);
       shell.isActive();
-      windowFound = true;
+      return true;
     } catch (WidgetNotFoundException exception) {
       throw new WrappedException("Error during searching for window", exception);
     }

```

Comment:
Do we not need a
```suggestion
    }
    return false;
```
here?

Reply:
I do not think so, that could would not be reachable, as either we return true or exit with the WrappedException.

Reply:
Oops, you're right, of course. I misread the code when I left the comment.